includes/sanitizers/class-amp-img-sanitizer.php

```diff
@@ -339,4 +340,36 @@ public function handle_centering( $node ) {
 
 		return $figure;
 	}
+
+	/**
+	 * Add an inline style to set the `<figure>` element's width to `auto` instead of `fit-content`.
+	 *
+	 * @since 1.0
+	 * @see https://github.com/Automattic/amp-wp/issues/1086
+	 *
+	 * @param DOMNode $node The DOMNode to adjust and replace.
```

Comment:
I think we should be able to safely replace `DOMNode` with `DOMElement` here. We already know it is an element because we just called it via `AMP_DOM_Utils::create_node()`.